appinventor/appengine/build.xml

```diff
@@ -360,16 +360,29 @@
     <delete file="${tmp.dir}/authkey/meta"/>
     <delete file="${tmp.dir}/authkey/1"/>
   </target>
-
+  <!-- =====================================================================
+       Check to see if the war/WEB-INF directory already has an
+       authkey, which would mean it was copied over in WarLibs from
+       the source tree. In the master branch this will *not* be the
+       case, but it may be in a branding branch.
+       ===================================================================== -->
+  <target name="CheckBuildAuthKey"
+          depends="init,CheckAuthKey,WarLibs">
```

Comment:
Since these dependencies precede `CheckBuildAuthKey` on line 380 we may not need to duplicate `CheckAuthKey,WarLibs` here. This would keep the signature parallel to `CheckAuthKey`.